src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+  flex-flow: wrap;
```

Comment:
Don't use shorthand property, if you don't benefit from it.
To set one rule - just use `flex-wrap`.